hoot-core/src/main/cpp/hoot/core/io/DataConverter.cpp

```diff
@@ -378,25 +377,9 @@ void DataConverter::_convert(const QString input, const QString output)
     LOG_VART(conf().get(ConfigOptions().getTranslationScriptKey()));
   }
 
```

Comment:
moved a bunch of this "is streamable" type logic from DataConverter to ElementStreamer